src/hooks/useStakingWidgetUpdate.ts

```diff
@@ -0,0 +1,27 @@
+import { useState, useEffect } from 'react';
+
+import { MotionStakes } from '~gql';
+import { SetStateFn } from '~types';
+
+import { compareMotionStakes } from './helpers';
+
+const useStakingWidgetUpdate = (
```

Comment:
When we stake successfully, we start polling the db again to get the updated staking values from the block ingestor. 

When the motion stakes object's content changes, we know to stop polling.